operations/before.go

```diff
@@ -105,9 +105,9 @@ var (
 		confPath := c.String("conf")
```

Comment:
You mentioned that for the BF involved, the path given was the empty string. Does that mean confPath is empty, and can we return early in that case? (i.e. just return no error at all)

Reply:
Yep, that should be fine